compare_floats.cpp

```diff
@@ -0,0 +1,71 @@
+#include <iostream>
```

Comment:
Try hexdump target in makefile. If keeping this merge it into arithmeticTest.cpp